### ISSUE TITLE
pip install fails at dependency psycopg2-binary==2.9.1

### ISSUE THREAD

**Issue by tsgouvea** [heartexlabs/label-studio#2879]
**Describe the bug**
Pip install label-studio fails to install dependency psycopg2-binary==2.9.1 

**To Reproduce**
1. Create new environment (e.g. conda)
2. run `pip install label-studio`
3. alternatively, clone repo and run pip install locally
4. Wait until `Collecting psycopg2-binary==2.9.1`
5. See error `Preparing metadata (setup.py) ... error` (long version below)

**Expected behavior**
Installs all dependencies and label-studio itself.

**Environment (please complete the following information):**
 - OS: macOS 12.3.1 (M1 chip)
 - Label Studio Version 1.5.0post0

**Full error log**
```sh
Collecting psycopg2-binary==2.9.1
  Using cached psycopg2-binary-2.9.1.tar.gz (380 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [25 lines of output]
      /path/to/miniforge3/envs/label-studio/lib/python3.10/site-packages/setuptools/config/setupcfg.py:508: SetuptoolsDeprecationWarning: The license_file parameter is deprecated, use license_files instead.
        warnings.warn(msg, warning_class)
      running egg_info
      creating /private/var/folders/dg/10ccgrxn55j86d9mjzbw94200000gn/T/pip-pip-egg-info-s9dk4hg0/psycopg2_binary.egg-info
      writing /private/var/folders/dg/10ccgrxn55j86d9mjzbw94200000gn/T/pip-pip-egg-info-s9dk4hg0/psycopg2_binary.egg-info/PKG-INFO
      writing dependency_links to /private/var/folders/dg/10ccgrxn55j86d9mjzbw94200000gn/T/pip-pip-egg-info-s9dk4hg0/psycopg2_binary.egg-info/dependency_links.txt
      writing top-level names to /private/var/folders/dg/10ccgrxn55j86d9mjzbw94200000gn/T/pip-pip-egg-info-s9dk4hg0/psycopg2_binary.egg-info/top_level.txt
      writing manifest file '/private/var/folders/dg/10ccgrxn55j86d9mjzbw94200000gn/T/pip-pip-egg-info-s9dk4hg0/psycopg2_binary.egg-info/SOURCES.txt'
      
      Error: pg_config executable not found.
      
      pg_config is required to build psycopg2 from source.  Please add the directory
      containing pg_config to the $PATH or specify the full executable path with the
      option:
      
          python setup.py build_ext --pg-config /path/to/pg_config build ...
      
      or with the pg_config option in 'setup.cfg'.
      
      If you prefer to avoid building psycopg2 from source, please install the PyPI
      'psycopg2-binary' package instead.
      
      For further information please check the 'doc/src/install.rst' file (also at
      <https://www.psycopg.org/docs/install.html>).
      
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

```

**Comment by tsgouvea** [comment]
P.S. Same error arises with different Python versions (3.8-10)

**Comment by stephanie-jauss** [comment]
Installing postgresql and openssl fixed it for me on a M2 MacBook Air (macOS 12.5). See https://github.com/psycopg/psycopg2/issues/1200

**Comment by makseq** [comment]
@tsgouvea please try to use `brew` on macos:

```
brew install heartexlabs/tap/label-studio
```

**Comment by jorgeorpinel** [comment]
Hi, https://labelstud.io/guide/install.html#Software-requirements states that your system needs PostgreSQL \*or\* SQLite, however (even if you plan to use the latter), PostgreSQL is always required?

Also, no mention of the Homebrew install option in docs. Is it experimental/ unofficial? Thanks

**Comment by makseq** [comment]
Hi, yes, postgres is required anyway. 

Thanks for pointing us about brew: it's not in the guide section. But it's on the main page. 
![image](https://user-images.githubusercontent.com/501780/225802241-33ea3281-c800-49c0-826c-71bc225fb0a9.png)

However, we should add it to the guide section too.

**Comment by Lauler** [comment]
@makseq 

What dependencies do I need to install label-studio? Failing to install on fresh ubuntu 22.04 install after installing postgresql and psycog2-binary. 

Can you provide an example of all (system) dependencies that need to be installed?

**Comment by cozeybozey** [comment]
I have the same issue on debian

Edit: 
Fixed it by doing: 
```
conda install psycopg2
```

**Comment by nba-ibme** [comment]
Having this same issue on Windows 10. Any update? I'm trying to use only pip so installing with conda isn't an option.

**Comment by ACSE-cr22** [comment]
So how could I solve it in Win11?

**Comment by safaa199945** [comment]
> So how could I solve it in Win11?

i have the same issue, did you solve it?

**Comment by nadinator** [comment]
I fixed it by using python v10. Anything below 11 should work, I think.

**Comment by avivleecohen** [comment]
I have Windows 11. I fixed it by installing PostgreSQL version 16.3 and Python 3.11.
